src/components/editor/feature-components/MetaformFieldRequiredComponent.tsx

```diff
@@ -0,0 +1,74 @@
+import { Checkbox, FormControlLabel, Stack, Typography } from "@mui/material";
+import { MetaformField } from "generated/client";
+import strings from "localization/strings";
+import React, { FC, useState } from "react";
+import { selectMetaform, setMetaformField } from "../../../features/metaform-slice";
+import { useAppSelector, useAppDispatch } from "app/hooks";
+
+/**
+ * Component properties
+ */
+interface Props {
+  setUpdatedMetaformField: (updatedMetaformField?: MetaformField) => void;
```

Comment:
Pass the debounced function as prop so no need to define then in each components. Same as in the other components